airflow/providers/amazon/aws/triggers/emr.py

```diff
@@ -242,7 +242,7 @@ def __init__(
         )
 
     def hook(self) -> AwsGenericHook:
-        return EmrContainerHook(self.aws_conn_id)
+        return EmrContainerHook(aws_conn_id=self.aws_conn_id)
```

Comment:
Nice catch, hooks are mixed with positional/keyword arguments, and this might lead a problem when we create Thick wrapper hook

https://github.com/apache/airflow/blob/633217c84217f925c4c0a92e0b24ba314b64281b/airflow/providers/amazon/aws/hooks/emr.py#L53-L58